src/spotted/data/user.py

```diff
@@ -24,18 +25,30 @@ class User:
     user_id: int
     private_message_id: int | None = None
     ban_date: datetime | None = None
+    mute_date: datetime | None = None
+    mute_expire_date: datetime | None = None
```

Comment:
we could use `Optional[datetime]` here

source https://stackoverflow.com/questions/51710037/how-should-i-use-the-optional-type-hint

Reply:
The modern approach (3.10+) is the one currently used: https://peps.python.org/pep-0604/

Reply:
As said by TendTo, we are using the modern approch. Nothing to do here.

Reply:
interesting